### PR TITLE
feat/PKGBUILD

### PR DESCRIPTION
**Pull request by JarbasAl**
https://github.com/wenLiangcan/pip2pkgbuild

`pip2pkgbuild ovos-X -p python -b python-ovos-X`

TODO https://github.com/marketplace/actions/publish-aur-package

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,24 @@
+pkgbase='python-ovos-config'
+pkgname=('python-ovos-config')
+_module='ovos-config'
+pkgver='0.0.7'
+pkgrel=1
+pkgdesc="OpenVoiceOS configuration tool"
+url="https://github.com/OpenVoiceOS/ovos-config"
+depends=('python')
```
Comment (AIIX) [inline PKGBUILD:8]: Package is missing the actual dependencies

Comment (JarbasAl) [reply PKGBUILD:8]: these would be dependencies already in AUR correct? so we need to start from ovos-utils and go from that one

but meanwhile if i understand right (just got started on docs) wouldn't line 23 `python setup.py install...` still install the packages? even if not best practice until we get other deps in the AUR

Comment (AIIX) [reply PKGBUILD:8]: There are build deps and runtime deps in arch pkgbuild, if a package requires build deps then those need to be available before setup.py itself runs

Comment (AIIX) [reply PKGBUILD:8]: these are pkgbuilds that I had generated for OVOS a while back, haven't been updated to the latest but should give an idea [ovos-core-pkgbuilds.tar.gz](https://github.com/OpenVoiceOS/ovos-config/files/11194718/ovos-core-pkgbuilds.tar.gz)

**Pull request closed without merging**